flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/decorators/FlinkMasterDeploymentDecorator.java

```diff
@@ -138,13 +144,21 @@ private Container createJobManagerContainer(
 				new ContainerPortBuilder().withContainerPort(flinkConfig.getInteger(RestOptions.PORT)).build(),
 				new ContainerPortBuilder().withContainerPort(flinkConfig.getInteger(JobManagerOptions.PORT)).build(),
 				new ContainerPortBuilder().withContainerPort(blobServerPort).build()))
-			.withEnv(
-				BootstrapTools.getEnvironmentVariables(ResourceManagerOptions.CONTAINERIZED_MASTER_ENV_PREFIX, flinkConfig)
-					.entrySet()
-					.stream()
-					.map(kv -> new EnvVar(kv.getKey(), kv.getValue(), null))
-					.collect(Collectors.toList()))
+			.withEnv(buildEnvForContainer(flinkConfig)).withEnvFrom()
```

Comment:
useless `withEnvFrom`?

Reply:
Aha, i forget to remove it and will fix it.